.ci/packaging.groovy

```diff
@@ -65,8 +65,29 @@ pipeline {
           options { skipDefaultCheckout() }
           steps {
             deleteDir()
-            gitCheckout(basedir: "${BASE_DIR}")
+            script {
+              if(isUpstreamTrigger()) {
+                try {
+                  copyArtifacts(filter: 'packaging.properties',
+                                flatten: true,
+                                projectName: "Beats/beats/${env.JOB_BASE_NAME}",
+                                selector: upstream(fallbackToLastSuccessful: true))
+                  def props = readProperties(file: 'packaging.properties')
+                  gitCheckout(basedir: "${BASE_DIR}", branch: props.COMMIT)
+                } catch(err) {
+                  // Fallback to the head of the branch as used to be.
+                  gitCheckout(basedir: "${BASE_DIR}")
```

Comment:
in case of error getting the artifact the build will continue, Is that correct?

Reply:
Yes, just wanted to keep the backward compatibility with the existing behaviour, which it's not ideal, but just in case something bad happens when fetching the artifacts from the upstream build, at least the behaviour will remain working